<robot name="bmw_x5">

  <link name="map"/>

  <joint name="map_to_car" type="fixed">
    <parent link="map"/>
    <child link="car_link"/>
    <origin xyz="0 0 0" rpy="0 0 0"/>
  </joint>

  <link name="car_link"/>

  <joint name="car_to_lidar" type="fixed">
    <parent link="car_link"/>
    <child link="lidar_link"/>
    <origin xyz="0.3 0 0.15" rpy="0 0 0"/>
  </joint>

  <link name="lidar_link"/>

  <joint name="car_to_camera" type="fixed">
    <parent link="car_link"/>
    <child link="camera_link"/>
    <origin xyz="1.2 0 1.5" rpy="0 0 0"/>
  </joint>

  <link name="camera_link"/>

  <joint name="car_to_imu" type="fixed">
    <parent link="car_link"/>
    <child link="imu_link"/>
    <origin xyz="0 0 0.1" rpy="0 0 0"/>
  </joint>

  <link name="imu_link"/>

</robot>

